src/components/Input/index.js

```diff
@@ -86,10 +86,10 @@ class Input extends Component {
     }
 
     return (
-      <label>
-        {label && <span className="db mb3 w-100">
-          {label}
-        </span>}
+      <label className="vtex-input">
+        {label && (
```

Comment:
Prevent empty tag:
```
{label &&
    <label className="vtex-input">
```

Reply:
@kevinch input is inside label tag. can't dont that!

Reply:
_can't do that_